WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -138,32 +150,36 @@ private extension AddAttributeOptionsViewController {
     /// Cells currently configured in the order they appear on screen
     ///
     func configure(_ cell: UITableViewCell, for row: Row, at indexPath: IndexPath) {
-        switch cell {
-        case let cell as TextFieldTableViewCell where row == .termTextField:
+        switch (row, cell) {
+        case (.termTextField, let cell as TextFieldTableViewCell):
             configureTextField(cell: cell)
-        case let cell as BasicTableViewCell where row == .selectedTerms:
-            configureOption(cell: cell)
-        case let cell as BasicTableViewCell where row == .existingTerms:
-            configureOption(cell: cell)
+        case (let .selectedTerms(name), let cell as BasicTableViewCell):
+            configureOption(cell: cell, text: name)
+        case (.existingTerms, let cell as BasicTableViewCell):
+            configureOption(cell: cell, text: "Work in Progress")
```

Comment:
This was changed because since row now have cases with associated types( `.selectedTerms(name: String)` ) we can't compare with with `==` anymore.